wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -550,6 +550,17 @@ public class ZMUserSession: NSObject {
         }
     }
 
+    private func observeOnEpochChangeIfNeeded() {
+        guard e2eiFeature.isEnabled,
+              let mlsService = self.syncContext.mlsService else {
```

Comment:
If you are update to with develop `mlsService is always available now so no need to fetch from the context.